### ISSUE TITLE
Camera Support

### ISSUE THREAD

**Issue by hobbyquaker** [hobbyquaker/homekit2mqtt#61]
implement the functionality of https://github.com/KhaosT/homebridge-camera-ffmpeg ?

**Comment by hobbyquaker** [comment]
Does anyone want this? Would be kinda "out of scope" regarding MQTT, but running a Homebridge in parallel only for the cameras (as I do today) isn't a "nice" solution either...

**Comment by mattsheffiel** [comment]
I would definitely like the option down the road but it isn't a top priority for me at the moment.

**Comment by NovaGL** [comment]
I would suggest a MQTT to RTSP Bridge.
So the MQTT topic lists the current feed address.
It would be good to support MJPEG as well if possible.

**Comment by lepuso** [comment]
Would be a really useful feature (even considering the out of scope nature). Running another Homebridge  instance is definitely not a good option.

**Comment by kevinshane** [comment]
Yes please , running another homebridge is painful at the moment

**Comment by raufis27** [comment]
Also interested. Thank you.

**Comment by jsponz** [comment]
+1

**Comment by lorenzorobbi** [comment]
I think is a good idea to develop camera support
+1

**Comment by kevinshane** [comment]
Please add camera support!! Pretty please :P Currently I have to run another homebridge instance in order to get the `homebridge-camera-ffmpeg` plugin running.. and it's eating a lot cpu and ram usage

**Comment by hobbyquaker** [comment]
Just integrated the homebridge-camera-ffmpeg plugin. Happy to hear if it works for you.

**Comment by jsponz** [comment]
@hobbyquaker Thanks a lot! I will check in during the weekend!

**Comment by hobbyquaker** [comment]
seems like ffmpeg is missing.

also i would suggest to create the config via the webui, the source config seems to be wrong, should start with -re or smth. see homebridge-camera-ffmpeg docs also.

**Comment by jsponz** [comment]
You were right!

Now it is working!

Thanks again!

**Comment by hobbyquaker** [comment]
had to do a major change in v0.9.11, the cameras are now published as accessories on their own (listening on individual ports), publishing them as bridged accessories broke the still image function (that is needed to get a picture in motion sensor notifications from the same room). That means you will have to pair the cameras again and assign them to a room.

**Comment by raufis27** [comment]
When I add accessory I see 2 categories - Camera and IP_Camera. What is difference between two? Also how do I configure motion detect notification ?

**Comment by hobbyquaker** [comment]
The Categories are the same, doesn't matter which one you choose. IP_CAMERA is included in hap-nodejs for compatibility reasons, I was just to lazy to remove it ;-)
To get motion notifications you just have to have a motion detector service with enabled notifications in the same room as the cam.

**Comment by raufis27** [comment]
Understood! Thank you.
